pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/UfoerOmregningEnslig.kt

```diff
@@ -0,0 +1,1070 @@
+package no.nav.pensjon.brev.maler.fraser
+
+import no.nav.pensjon.brev.api.model.Kroner
+import no.nav.pensjon.brev.model.format
+import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
+import no.nav.pensjon.brev.template.Language.*
+import no.nav.pensjon.brev.template.OutlinePhrase
+import no.nav.pensjon.brev.template.dsl.expression.*
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brev.template.dsl.textExpr
+import java.time.LocalDate
+
+val VedtakOverskriftPesys_001 = OutlinePhrase<LangBokmalNynorskEnglish, Unit> {
+    title1 {
+        text(
+            Bokmal to "Vedtak",
+            Nynorsk to "Vedtak",
+            English to "Decision"
+        )
+    }
+}
+
+data class OmregnUTDodEPSInnledn1001Dto(val avdod_navn: String, val krav_virkedato_fom: LocalDate)
+
+val OmregnUTDodEPSInnledn1_001 = OutlinePhrase<LangBokmalNynorskEnglish, OmregnUTDodEPSInnledn1001Dto> {
```

Comment:
Burde vi vurdere å slå sammen disse `"Vi har mottat melding om at ..."` frasene til en frase som tar inn data som trengs for å velge riktig tekst?

Reply:
Ikke helt enig. Vi kan ta opp igjen diskusjonen senere.

Reply:
Var større forskjell på dem enn jeg trodde. Jeg trodde det var slik som i UngUfoer hvor det kun var listen av tillegg som var forskjell fra frase til frase. Se `no.nav.pensjon.brev.maler.fraser.omregning.ufoeretrygd.Ufoeretrygd#getBeloep`

Reply:
Slo sammen if-ene for innledningen. Tekstene beholdes som separate.